.github/workflows/test-pr.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install pytest pytest-cov pytest-asyncio discord chatgpt
+        pip install pytest pytest-cov pytest-asyncio discord chatgpt emoji
```

Comment:
Same here, remove chatgpt 
```suggestion
        pip install pytest pytest-cov pytest-asyncio discord emoji
```

Reply:
I think doing that would break the tests again. Imo, it's a separate PR to remove the gpt cog completely

Reply:
Fair enough